Signal/src/ViewControllers/SettingsTableViewController.m

```diff
@@ -186,6 +191,13 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
                     [self.navigationController pushViewController:vc animated:YES];
                     break;
                 }
+                case kBlockListRow: {
```

Comment:
On Android this lives within Privacy. For ease of support I think we should be consistent.